class Eh{
	public static void main(String args[]){
		System.out.println("Start");
		int a=100/20;
		System.out.println(a);
		System.out.println("end");
	}
}
		

//Exception opccured problem
class Eh{
	public static void main(String args[]){
		System.out.println("Start");
		int a=100/0;
		System.out.println(a);
		System.out.println("end");
	}
}
		
//output
Start
Exception in thread "main" java.lang.ArithmeticException: / by zero
        at Eh.main(Eh.java:4)

//Handling of Exception
class Eh{
	public static void main(String args[]){
		try{
			int a=100/0;
			System.out.println(a);
		}
		catch(Exception e){
			System.out.println("Test:::::  "+e);
		}finally{
			System.out.println("finally block");
		}

	}
}
		
//
